### ISSUE TITLE
config.py forces trailing : in remote names

### ISSUE THREAD

**Issue by jpcuzzourt** [ConorWilliams/rsinc#6]
During the interactive configuration, if the desired remote base is "mydropbox:rsinc", config.py adds a traling colon, making it "mydropbox:rsinc:"
I believe this may be in error. Or maybe I'm misusing BASE_R. Is it never intended that it should point to a directory on the remote? 

Edit:nevermind - I found my user error...

**Comment by ConorWilliams** [comment]
Yup this is a bug, thanks for letting me know. Rsinc should support sub directories of remote as BASE_R (as it does for BASE_L) should be a simple fix in config.py from: BASE_R ending with ':' to BASE_R contains ':'. will close issue once it's fixed :)

**Comment by jpcuzzourt** [comment]
Great. That's what it looked like to me. I'm just getting my feet wet with both rclone and rsinc, so I'm a little uncertain on some things, but plowing ahead anyway.
So far, I'm really appreciating what you've done with rsinc.
Thanks!
